packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_wallets_credit_transactions.py

```diff
@@ -0,0 +1,98 @@
+""" Wallets credit transaction
+    - Basically this table is balance sheet of each wallet.
+"""
+import enum
+
+import sqlalchemy as sa
+
+from ._common import column_modified_datetime
+from .base import metadata
+
+
+class TransactionBillingStatus(str, enum.Enum):
+    PENDING = "PENDING"
+    BILLED = "BILLED"
+    NOT_BILLED = "NOT_BILLED"
+    MANUAL_REVIEW = "MANUAL_REVIEW"
+
+
+class TransactionClassification(str, enum.Enum):
+    ADD_WALLET_TOP_UP = "ADD_WALLET_TOP_UP"  # user top up credits
+    DEDUCT_SERVICE_RUN = (
+        "DEDUCT_SERVICE_RUN"  # computational/dynamic service run costs)
+    )
```

Comment:
I would go for `DEDUCT_FOR_SERVICES` or `BILL_FOR_SERVICES` a generic plural since they could be billed for disk usage or something else in the future.

Reply:
Yes thats the thing, if the pricing will be connected to running of the service then SERVICE_RUN makes sense for me. If in the future there will be some other "type" then we will add new Transaction classification, for example: DEDUCT_STORAGE.